utils/sample_qc.py

```diff
@@ -364,91 +364,109 @@ def infer_sex(
-                (chr_x_ploidy < xx_x_ploidy_cutoffs[1]) &
-                (chr_y_ploidy < xx_y_ploidy_cutoff) &
-                (f_stat < f_stat_female_cutoff),
-                'XX')
+                ((ht.chrY_ploidy < xx_y_ploidy_cutoff) &
+                (ht.chrX_ploidy < xy_x_ploidy_cutoff)),
+                'X0')
                 .when(
-                (chr_x_ploidy > xy_x_ploidy_cutoffs[0]) &
-                (chr_x_ploidy < xy_x_ploidy_cutoffs[1]) &
-                (chr_y_ploidy > xy_y_ploidy_cutoffs[0]) &
-                (chr_y_ploidy < xy_y_ploidy_cutoffs[1]) &
-                (f_stat > f_stat_male_cutoff),
+                ((ht.chrX_ploidy < xy_x_ploidy_cutoff) &
+                (ht.chrY_ploidy > xy_y_ploidy_cutoffs[0]) &
+                (ht.chrY_ploidy < xy_y_ploidy_cutoffs[1])),
                 'XY')
                 .when(
-                (chr_x_ploidy > xy_x_ploidy_cutoffs[0]) &
-                (chr_x_ploidy < xy_x_ploidy_cutoffs[1]) &
-                (chr_y_ploidy > yy_y_ploidy_cutoff) &
-                (f_stat > f_stat_male_cutoff),
+                ((ht.chrX_ploidy < xy_x_ploidy_cutoff) &
+                (ht.chrY_ploidy >= yy_y_ploidy_cutoff)),
                 'XYY')
                 .when(
-                (chr_x_ploidy > xx_x_ploidy_cutoffs[0]) &
-                (chr_x_ploidy < xx_x_ploidy_cutoffs[1]) &
-                (chr_y_ploidy > xy_y_ploidy_cutoffs[0]) &
-                (chr_y_ploidy < xy_y_ploidy_cutoffs[1]) &
-                (f_stat < f_stat_female_cutoff),
+                ((ht.chrX_ploidy > xx_x_ploidy_cutoffs[0]) &
+                (ht.chrX_ploidy < xx_x_ploidy_cutoffs[1]) &
+                (ht.chrY_ploidy < xx_y_ploidy_cutoff)),
+                'XX')
+                .when(
+                ((ht.chrX_ploidy > xx_x_ploidy_cutoffs[0]) &
+                (ht.chrX_ploidy < xx_x_ploidy_cutoffs[1]) &
+                (ht.chrY_ploidy > xy_y_ploidy_cutoffs[0]) &
+                (ht.chrY_ploidy < xy_y_ploidy_cutoffs[1])),
                 'XXY')
                 .when(
-                (chr_x_ploidy > xxx_x_ploidy_cutoff) &
-                (chr_y_ploidy < xx_y_ploidy_cutoff) &
-                (f_stat < f_stat_female_cutoff),
-                'XXX')
+                ((ht.chrX_ploidy > xx_x_ploidy_cutoffs[0]) &
+                (ht.chrX_ploidy < xx_x_ploidy_cutoffs[1]) &
+                (ht.chrY_ploidy >= yy_y_ploidy_cutoff)),
+                'XXYY') 
                 .when(
-                (chr_y_ploidy < xx_y_ploidy_cutoff) &
-                (chr_x_ploidy > xy_x_ploidy_cutoffs[0]) &
-                (chr_x_ploidy < xy_x_ploidy_cutoffs[1]) &
-                (f_stat > f_stat_male_cutoff),
-                'X0')
-                .default('Ambiguous')
-        )
+                ((ht.chrX_ploidy >= xxx_x_ploidy_cutoff) &
+                (ht.chrY_ploidy < xx_y_ploidy_cutoff)),
+                'XXX') 
+                .default('Ambiguous') # NOTE: XXXY, XXXYY, XYYY, XXXX, XXXXY, XXXXX also exist 
```

Comment:
We could rename this to `Other` if you think it fits better?

Reply:
I like ambiguous, since samples that fall just outside the cutoffs will fall into this default bucket too, but happy to change this to any other terms

Reply:
Ambiguous is fine by me -- this was just about the note you added